src/routes/[...lang=locale]/anime/+page.svelte

```diff
@@ -329,51 +329,65 @@
 					{#each latest_episodes as episode}
-						{@const has_color_palette = latest_episodes_color_palette_mapping[episode.id] !== undefined}
-						{@const color_palette = has_color_palette && rgbHex(...latest_episodes_color_palette_mapping[episode.id][0])}
+						{@const color_palette =
+							latest_episodes_color_palette_mapping[episode.id] !== undefined &&
+							rgbHex(...latest_episodes_color_palette_mapping[episode.id][0])}
+						{@const image_loaded = latest_episodes_loaded_mapping[episode.id]}
 
-						<div
-							class="snap-start w-full md:h-[5vw] bg-cover bg-center relative md:rounded-[0.75vw] border md:border-[0.15vw] border-accent/50 duration-300"
-							style="
+						{#if image_loaded}
+							<div
+								class="relative w-full snap-start border border-accent/50 bg-cover bg-center duration-300 md:h-[5vw] md:rounded-[0.75vw] md:border-[0.15vw]"
+								style="
 								background-image: url({episode.banner});
 								border-color: #{color_palette};
 							"
-						>
-							<div class="absolute inset-0 bg-secondary/75 md:rounded-[0.75vw]"></div>
-							<div class="relative size-full flex items-center md:p-[0.5vw] md:gap-[1vw]">
-								<img
-									src={episode.cover}
-									alt=""
-									class="md:w-[2.5vw] h-full object-cover object-center md:rounded-[0.5vw]"
-								/>
-								<div class="flex flex-col flex-1 md:gap-[0.15vw]">
-									<span class="text-accent md:text-[1.15vw] font-bold line-clamp-1">{episode.title}</span>
-									<div class="md:leading-none flex items-center md:gap-[0.5vw] md:text-[0.8vw] font-semibold">
-										<span class="whitespace-nowrap">Ep {episode.ep_number.toString().padStart(2, "0")}</span>
-										<Circle class="md:size-[0.25vw] opacity-75" />
-										<span class="line-clamp-1">{episode.timestamp}</span>
+							>
+								<div class="absolute inset-0 bg-secondary/75 md:rounded-[0.75vw]"></div>
+								<div class="relative flex size-full items-center md:gap-[1vw] md:p-[0.5vw]">
+									<img
+										src={episode.cover}
+										alt=""
+										class="h-full object-cover object-center md:w-[2.5vw] md:rounded-[0.5vw]"
+									/>
+									<div class="flex flex-1 flex-col md:gap-[0.15vw]">
+										<span class="line-clamp-1 font-bold text-accent md:text-[1.15vw]"
+											>{episode.title}</span
+										>
+										<div
+											class="flex items-center font-semibold md:gap-[0.5vw] md:text-[0.8vw] md:leading-none"
+										>
+											<span class="whitespace-nowrap"
+												>Ep {episode.ep_number.toString().padStart(2, "0")}</span
+											>
+											<Circle class="opacity-75 md:size-[0.25vw]" />
+											<span class="line-clamp-1">{episode.timestamp}</span>
+										</div>
 									</div>
+									<a
+										href="anime/mal/{episode.id}/episode/{episode.ep_number}"
+										class="btn h-max min-h-max rounded-full border-none bg-warning md:mr-[0.5vw] md:p-[0.75vw]"
+										style="background-color: #{color_palette};"
```

Comment:
Avoid using direct style and use var() with tailwind here

Reply:
Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/77/commits/58859ef6cf58e42a95267d9bad6f91b93a3234ee